src/ui/ui_render/ui_render_mixin.js

```diff
@@ -139,7 +139,7 @@ export function uiRenderMixin(kbnServer, server, config) {
     try {
       const request = reply.request;
       const translations = await server.getUiTranslations();
-      const basePath = config.get('server.basePath');
+      const basePath = request.getBasePath();
```

Comment:
Would you mind explaining why we're using the space relative base path here as opposed to the root path?